src/client/datascience/notebookStorage/baseModel.ts

```diff
@@ -230,12 +223,14 @@ export abstract class BaseNotebookModel implements INotebookModel {
     protected _isDisposed?: boolean;
     protected _changedEmitter = new EventEmitter<NotebookModelChange>();
     protected _editEventEmitter = new EventEmitter<NotebookModelChange>();
+    protected _kernelConnection?: KernelConnectionMetadata;
     private kernelId: string | undefined;
```

Comment:
I think you can get rid of this now? Is'nt the storage for this in that other location now?

Reply:
Yes, but the other location inherits this class. Hence is required.

Reply:
Not the _kernelConnection, the kernelId.

Reply:
Yup, removed, thanks